README.md

```diff
@@ -16,54 +16,306 @@ Automatically recompile css file on change.
 
 ### Run the application
 
-You can run from the command line, or from within the Cursive Clojure IDE
-(recommended).
+> Henceforth, for clarity and accuracy, we'll assume you're on MacOS. The use
+> of other platforms with these tools is certainly possible, and the process is
+> probably quite similar to the exposition below, but it is currently untested.
 
-##### From the command line
+To run Owlet UI as a developer, you must start it from the command line in a
+terminal. In what follows, we'll assume the current directory is the Owlet UI
+project directory (the one containing this README.md file), so `cd` there and
+run the following command: (This assumes you have `lein` and `rlwrap`
```

Comment:
Until i get around to adding the link to getting started with clojure from http://www.clojurebridge.org/ can we add links to `lein` and `rlwrap` ?